generatebundlefile/scripts/setup.sh

```diff
@@ -0,0 +1,27 @@
+#!/usr/bin/env bash
+# Copyright Amazon.com Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -x
+set -e
+set -o pipefail
+
+USR_BIN=/usr/bin
+
+if [ ! -d "/root/.docker" ]; then
```

Comment:
Where is this file run? Why is it assuming `/root/.docker`?

Reply:
This is run by the docker daemon, just copied it from build-tooling TBH

Reply:
This is used by the docker daemon config, copied from the eks-build-tooling how they handle container/helm builds.